fixtures/component_fixtures/common/EC2UnsupportedVersion/component.yaml

```diff
@@ -14,7 +14,7 @@ satisfies:
   implementation_status: partial
   narrative: Justification in narrative form
   standard_key: NIST-800-53
-schema_version: 1.0
+schema_version: 1.0.0
```

Comment:
Do we think there are schemas out there that have `1.0` listed? If so, will we still support that?